app/user_config.js

```diff
@@ -1,15 +1,31 @@
 const path = require('path');
+const process = require('process');
 
 const { app } = require('electron');
 
 const { start } = require('./base_config');
 const config = require('./config');
 
-// Use separate data directory for development
-if (config.has('storageProfile')) {
+let storageProfile;
+const { NODE_ENV: environment, NODE_APP_INSTANCE:instance } = process.env;
+const isValidInstance = instance && instance.length > 0;
```

Comment:
Smelled a little weird but node make sure environment variables are always strings. Maybe good to note in the code because we'd throw on the .length if it's not a string

Reply:
Good point, addressed in the code :)